src/components/header/header.module.css

```diff
@@ -0,0 +1,122 @@
+.logo{
+    width: 110px;
+    height: 70px;
+}
+.navbarContainer{
+    background: #FFFFFF;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    padding: 0;
+}
+.menuButton{
+  border: none;
+  background-color: #FFFFFF;
+}
+.toggleIcon{
+  font-size: 15px;
+  border: 1px solid #000000;
+  padding: 0.5em;
+  border-radius: 10px;
+}
+.navigationMenu{
+  display: flex;
+  justify-content: flex-end !important;
+}
+.menuItems{
+  margin-right: 1rem;
+  margin-left: 1rem;
+  text-decoration: none;
+  color: #000000;
+  font-size: 16px;
+  font-weight: 400;
+}
```

Comment:
It would be great if this <a> elements where width 100%. It would be easier for users to click them